### PR TITLE
Fix CS

### PR DESCRIPTION
**Pull request by szepeviktor**
Managed to reduce the number of wpcs-tsf violations.

### CODE REVIEW

File: phpcs.xml [hunk 1]
```diff
@@ -27,6 +27,14 @@
 		</properties>
 	</rule>
 
+	<rule ref="WordPress.Security.EscapeOutput">
+		<properties>
+			<property name="customEscapingFunctions" type="array">
+				<element value="esc_xml"/>
```
Comment (sybrew) [inline phpcs.xml:33]: Should this not be added upstream?
https://developer.wordpress.org/reference/functions/esc_xml/

**Comment by szepeviktor** [comment]
@sybrew 🛑 Stop!

Seems like `_init_post_edit_view` is defined in one class and hooked in another class.

File: phpcs.xml (hunk 1 above)
Comment (szepeviktor) [reply phpcs.xml:33]: You talk to Juliette.
https://github.com/WordPress/WordPress-Coding-Standards/issues/1937

**Comment by sybrew** [comment]
Most callbacks are hooked in the `init.class.php`. I'm still figuring out how to manage this better; this probably must be done procedurally, as WordPress does.

About the PR: On most of the functions altered, I have left the extraneous parameters for the future: "might need, don't forget these exist."

I'm on the fence about removing them. Merging would improve performance and make things neater overall.

**Comment by szepeviktor** [comment]
All right.

> "might need, don't forget these exist."

These could go into GitHub issues!

**Comment by sybrew** [comment]
Or as a comment in the function?

**Comment by szepeviktor** [comment]
> Most callbacks are hooked in the `init.class.php`. I'm still figuring out how to manage this better

Use my annotation based hooking.
```php
/**
 * @hook plugins_loaded
 */
public function init()
{
```
It is so tiny: https://github.com/szepeviktor/SentencePress/blob/master/src/HookAnnotation.php

**Comment by szepeviktor** [comment]
> Or as a comment in the function?

Nooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo!
TODO-s and FIXME-s were in the code back then when patches where send through email.

**Comment by sybrew** [comment]
Following up on https://github.com/sybrew/the-seo-framework/pull/642#issuecomment-1610257514
> Most callbacks are hooked in the `init.class.php`. I'm still figuring out how to manage this better; this probably must be done procedurally, as WordPress does.

In the past years, I've also been trying to dilute (thin out) the god-object `tsf()` in the past years by moving methods to their own objects. Ref https://github.com/sybrew/the-seo-framework/issues/166 and https://github.com/sybrew/the-seo-framework/issues/583.

Re https://github.com/sybrew/the-seo-framework/pull/642#issuecomment-1610260322: I like the trait and I think it's genius, but it conflicts with my goal to achieve the highest performance.

File: phpcs.xml (hunk 1 above)
Comment (sybrew) [reply phpcs.xml:33]: WPCS 3.0 appears to have implemented this now. 

I'll implement this workaround and remove it once I upgrade wpcs-tsf for TSF v4.3.1+.

Comment (szepeviktor) [reply phpcs.xml:33]: All right.

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**